src/test/fio/ceph-librgw.fio

```diff
@@ -0,0 +1,28 @@
+#
+# example jobfile, e.g.:
+# fio --max-jobs=20  /lv2tb/ceph-cp/src/test/fio/ceph-librgw.fio
+#
+[global]
+ioengine=external:/home/mbenjamin/ceph-cp/build/lib/libfio_librgw.so
+name=fiotest
+direct=0
+access_key=${AWS_ACCESS_KEY_ID}
+secret_key=${AWS_SECRET_ACCESS_KEY}
+userid=testuser
+ceph_cluster=ceph
```

Comment:
the `userid` and `ceph_cluster` values are not enforced, mistakingly misconfigured and workload still went thru to run OK.

Reply:
@mkogan1 ok, thanks, Mark;  I'll revisit in the next PR